src/docsfy/api/projects.py

```
@@ -0,0 +1,1592 @@
+    # Always pass current_stage through so that None clears the stage in the DB.
+    ups_kwargs["current_stage"] = current_stage
+
+    await update_project_status(project_name, ai_provider, ai_model, **ups_kwargs)
+
+    if status in _TERMINAL_STATUSES:
+        await notify_status_change(
+            gen_key=gen_key,
+            status=status,
+            page_count=page_count,
+            last_generated=(
+                datetime.now(timezone.utc).strftime("%Y-%m-%d %H:%M:%S")
```

Ruff (0.15.6)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)